packages/nep141-erc20/src/bridged-nep141/sendToEthereum/index.js

```diff
@@ -575,16 +582,23 @@ async function unlock (transfer) {
   const borshProof = borshifyOutcomeProof(last(transfer.proofs))
   const nearOnEthClientBlockHeight = new BN(last(transfer.nearOnEthClientBlockHeights))
 
+  // If this tx is dropped and replaced, lower the search boundary
+  // in case there was a reorg.
+  let safeHeightBeforeEthTx = await web3.eth.getBlockNumber() - 20
```

Comment:
When a transaction is dropped and replaced, the replacement tx will be searched. But the replacement transaction doesn't necessarily land in the same block number. So in the unlikely scenario of a chain reorg happens at the same time, we should widen the search scope and search 20 blocks before as a precaution otherwise we risk not finding the replacement tx.

Reply:
Does this whole approach assume the page is open the whole time during a transfer? Is that ok?

Reply:
No, the user can come back.